packages/mobile/src/geth/networkConfig.ts

```diff
@@ -17,50 +17,72 @@ interface NetworkConfig {
   syncMode: GethSyncMode
   initiallyForno: boolean
   blockchainApiUrl: string
+  pgpnpUrl: string // Phone Number Privacy service url
+  pgpnpPubKey: string
 }
 
+const pgpnpUrlStaging = 'https://us-central1-celo-phone-number-privacy-stg.cloudfunctions.net'
+const pgpnpPubKeyStaging =
+  'B+gJTCmTrf9t3X7YQ2F4xekSzd5xg5bdzcJ8NPefby3mScelg5172zl1GgIO9boADEwE67j6M55GwouQwaG5jDZ5tHa2eNtfC7oLIsevuUmzrXVDry9cmsalB0BHX0EA'
+const pgpnpUrlProd = 'https://us-central1-celo-phone-number-privacy.cloudfunctions.net'
+const pgpnpPubKeyProd = 'TODO'
```

Comment:
Remove TODO string or leave comment explaining?

Reply:
I'll be updating this before merging :)